api/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
 
-# Create your models here.
+ACTIVITY_LEVEL = (
+			("SC", "Sleepy_Cat"),
+			("LC", "Lazy_Cat"),
+			("AC", "Active_Cat"),
+			("MC", "Mentor_Cat"),
+	)
+
+class UserProfile(models.Model):
+	user = models.OneToOneField(User) 
```

Comment:
What is the meaning of the field user? You already have the primary key "id" in the next line (line 14).